src/routers/comment.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { commentController } from '../controllers';
+import { isAuth } from '../middlewares/isAuth';
+import { checkUserDoctor } from '../utils/checkRole';
+
+const router = express.Router({ mergeParams: true });
```

Comment:
ở chỗ này vì răng lại add thêm option mergeParams vậy Long

Reply:
mergeParams sử dụng để merge vào question routes.

Reply:
cơ chế chỗ này như răng á Long, t chưa biết cái ni nên chưa hiểu lắm

Reply:
cơ chế là route này t dựa vào route question, chứ k route riêng cho nó.
ví dụ: api/question/answer. chứ không phải là:
api/question
api/answer

Reply:
à okay, thank you